feature-libs/requested-delivery-date/core/http-interceptors/bad-request/requested-delivery-date-badrequest.handler.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpErrorResponse, HttpRequest } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import {
+  ErrorModel,
+  GlobalMessageType,
+  HttpErrorHandler,
+  HttpResponseStatus,
+  Priority,
+} from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RequestedDeliveryDateBadRequestHandler extends HttpErrorHandler {
```

Comment:
Hi @vigneshkrish8691,

As per the coding standards, we use the name convention in such a manner...

`<Type OF Error>-Handler`. This will help us identify easily the requests that would be handled by this class.

Please find similar occurrences at -

https://github.com/SAP/spartacus/blob/ce69fafafffed31e228c05a614e13bbc712aac53/projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts#L20

https://github.com/SAP/spartacus/blob/ce69fafafffed31e228c05a614e13bbc712aac53/projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts#L20

https://github.com/SAP/spartacus/blob/ce69fafafffed31e228c05a614e13bbc712aac53/feature-libs/organization/administration/core/http-interceptors/bad-request/bad-request.handler.ts#L20

https://github.com/SAP/spartacus/blob/ce69fafafffed31e228c05a614e13bbc712aac53/feature-libs/checkout/b2b/core/http-interceptors/bad-request/bad-cost-center-request.handler.ts#L22


There are a couple of others as well.

I believe we had this discussion previously.